server/commands/write_authzmodel.go

```diff
@@ -35,12 +35,12 @@ func (w *WriteAuthorizationModelCommand) Execute(ctx context.Context, req *openf
 		return nil, serverErrors.ExceededEntityLimit("type definitions in an authorization model", w.backend.MaxTypesInTypeDefinition())
 	}
 
-	schemaVersion, err := typesystem.NewSchemaVersion(req.GetSchemaVersion())
+	version, err := typesystem.NewSchemaVersion(req.GetSchemaVersion())
```

Comment:
why this change? shcemaVersion is clear

Reply:
The function call lines are getting so long and I thought is was pretty clear because in the write auth model command, but I'll revert it for now.